document/CoreFormat.mdk

```diff
@@ -11,8 +11,10 @@ that is, it shall consist of a comma-separated sequence of name/value pairs, enc
 as described in the JSON specification.
 We refer to the object represented by this top-level value as the `sarifLog` object (&sect;[#sarifLog]).
 
-Every JSON property name in the SARIF format shall be a camelCase name.
-This includes the names of properties defined in property bags such as `result.properties` (&sect;[#result-properties]).
+Every JSON property name defined by the SARIF format shall be a camelCase name.
+Because the names of properties defined in property bags (&sect;[#property-bag-properties]) such as `result.properties` (&sect;[#result-properties]),
+are not defined by the SARIF format, they are not subject to this requirement.
+These property names should also be cameCase, but see [Annex C](#Annex-C) for exceptions.
```

Comment:
That guidance is in Annex C, which I checked in previously. The reason for this change is that the main body of the spec, this text here, was inconsistent with the new Annex C.